PennMobile/General/Networking + Analytics/UserDBManager.swift

```diff
@@ -32,6 +32,23 @@ class UserDBManager: NSObject, Requestable {
         let task = URLSession.shared.dataTask(with: request as URLRequest, completionHandler: callback)
         task.resume()
     }
+    
+    fileprivate func makePostRequestWithAccessToken(url: String, params: [String: String], callback: @escaping (Data?, URLResponse?, Error?) -> Void) {
```

Comment:
Can you add a line comment above this function describing it? Like when it should be used vs. regular post requests, etc

Reply:
Sure.